motors/dc_motor_class.c

```diff
@@ -112,7 +112,7 @@
  * .
  * `time_sp`
  * : (read/write) Sets the time setpoint used with the `run-timed` command.
- *   Units are in milliseconds.
+ *   Units are in milliseconds. Values must not be negative.
```

Comment:
No worries. In many cases I have checked. I will do an exhaustive check tonight (currently apt upgrading and then going to work).